src/SerilogWeb.Classic/Classic/ApplicationLifecycleModule.cs

```diff
@@ -36,6 +37,7 @@ public class ApplicationLifecycleModule : IHttpModule
         static volatile IEnumerable<string> _filteredKeywords = new[] { "password" };
         static volatile LogEventLevel _requestLoggingLevel = LogEventLevel.Information;
         static volatile LogEventLevel _formDataLoggingLevel = LogEventLevel.Debug;
+        static volatile IList<Func<HttpContext, bool>> _requestPredicates = new List<Func<HttpContext, bool>>();
```

Comment:
If we're going to allow on-the-fly adding of predicates, needs to be something like a `ConcurrentBag<>`.